src/main/java/mate/academy/bookshop/dto/BookDto.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.bookshop.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
```

Comment:
why would we need to mark DTO classes with a `@Component`?
Components are usually services, util classes, and other stuff like this, but never a POJO/DTO classes